staging/src/k8s.io/kubectl/pkg/cmd/create/create_token.go

```diff
@@ -100,6 +100,7 @@ var (
 	boundObjectKindToAPIVersion = map[string]string{
 		"Pod":    "v1",
 		"Secret": "v1",
+		"Node":   "v1",
```

Comment:
Happy to make it so - are there other examples of how this is done/the naming scheme for it? Is the error returned from the apiserver if this feature is disabled not sufficient communication to users? (or do we specifically want local clients to opt-in to alpha behaviour too, even if the server enables it already?)

Reply:
Clients should opt-in to using alpha behavior in some way.  With REST APIs this is easy because the API itself has a version in it.  For this command I think something like https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/3895-kubectl-delete-interactivity#alpha would be sufficient.

Reply:
done